controllers/mover/syncthing/tlsutils.go

```diff
@@ -0,0 +1,180 @@
+package syncthing
```

Comment:
The functions in this file seem to be very similar. Is there a way to consolidate the duplicate code?

Reply:
Yep, I pushed a commit that reduces the bloat in this file.